test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/BasicWebSite.Area1.RemoteAttribute_Home.Create.html

```diff
@@ -83,10 +91,10 @@
 
     </div>
 
-    <script src="/lib/jquery/jquery.js"></script>
+    <script src="/javascript/jquery-2.1.4.min.js"></script>
```

Comment:
Ah it's not published there I guess. I don't mind checking in our own files, which are fine license-wise, and also not very large. But the huge 3rd party stuff we shouldn't include.